packages/passport-interface/src/PassportInterface.ts

```diff
@@ -15,6 +15,7 @@ export interface PCDRequest {
 
 export interface ProveOptions {
   genericProveScreen?: boolean;
+  iconSrc?: string;
```

Comment:
worth naming this something more verbose - eg. `headerIconImageSrc`. also I think this + the rest of the fields in `ProveOptions` should probably have comments, though maybe we can leave that for later if you don't want to do it here.